app/SQL/install.sql.mysql.php

```diff
@@ -27,6 +27,7 @@
 	`lastUpdate` INT(11) DEFAULT 0,	-- Until year 2038
 	`priority` TINYINT(2) NOT NULL DEFAULT 10,
 	`pathEntries` VARCHAR(511) DEFAULT NULL,
+	`pathEntriesFilter` VARCHAR(511) DEFAULT NULL,
```

Comment:
Please save this information in the JSON `attribute` column instead. I would rather avoid a new column for this type of data. See example of `xpath`